lib/ls-js-sdk.js

```diff
@@ -9,9 +8,9 @@ var glucose = require("ls-glucose");
 /**
   * (Public) Function that initialises the SDK setting the configuration object
   * @param config {Object} Configuration object, which accepts following properties:
-  *   app_id {String} An API ID (required)
-  *   app_key {String} An API Key (required)
-  *   api_user {String} User ID (optional)
+  *   app_token {String} An authentication token
+  *   app_id {String} An API ID (required) DEPRECATED
```

Comment:
My thinking was they will be deprecated once we move to a generated token functionality. But I agree there might be use cases when such configuration is sufficient. I'll leave it to @webpolk to decide.